src/course-outline/utils.jsx

```diff
@@ -25,6 +25,10 @@ const getItemStatus = ({
     return ITEM_BADGE_STATUS.gated;
   case visibilityState === VisibilityTypes.LIVE:
     return ITEM_BADGE_STATUS.live;
+  case visibilityState === VisibilityTypes.UNSCHEDULED:
+    return ITEM_BADGE_STATUS.unscheduled;
+  case visibilityState === VisibilityTypes.NEEDS_ATTENTION && !hasChanges:
```

Comment:
@kaustavb12 I cannot remember the scenario where needs attention was true and has changes was false. I had to include it before `case published && !hasChanges` as published was true and it would be caught before the default draft state.

I am removing it for now as I am not able to create a scenario where this could be true. We can add it back in future if required.